node/src/app/Products/Products.js

```diff
@@ -173,16 +170,40 @@ class HomePage extends Component {
                                         </Grid>
                                     ))}
                                 </Grid>
-                                {renderPageNumbers}
+                                {
+                                    _.range(1, totalPages + 1).map(number => {
```

Comment:
renamer `number` to a more explicit name. ex. `page` or `pageNumber`.

Reply:
Done